app/viewModels/document/DocumentAnswersViewModel.scala

```diff
@@ -29,28 +30,39 @@ object DocumentAnswersViewModel {
 
   class DocumentAnswersViewModelProvider @Inject() () {
 
-    def apply(userAnswers: UserAnswers, index: Index)(implicit messages: Messages): DocumentAnswersViewModel = {
+    def apply(userAnswers: UserAnswers, index: Index)(implicit messages: Messages, phaseConfig: PhaseConfig): DocumentAnswersViewModel = {
       val helper = new DocumentAnswersHelper(userAnswers, index)
 
-      val section = Section(
-        rows = Seq(
-          helper.attachToAllItems,
-          helper.documentType,
-          helper.previousDocumentType,
-          helper.documentReferenceNumber,
-          helper.typeOfPackageYesNo,
-          helper.packageType,
-          helper.numberOfPackagesYesNo,
-          helper.numberOfPackage,
-          helper.declareQuantityOfGoodsYesNo,
-          helper.metric,
-          helper.quantity,
-          helper.lineItemNumberYesNo,
-          helper.lineItemNumber,
-          helper.additionalInformationYesNo,
-          helper.additionalInformation
-        ).flatten
-      )
+      val rows = Seq(
+        helper.attachToAllItems,
+        helper.documentType,
+        helper.previousDocumentType,
+        helper.documentReferenceNumber,
+        helper.typeOfPackageYesNo,
+        helper.packageType,
+        helper.numberOfPackagesYesNo,
+        helper.numberOfPackage,
+        helper.declareQuantityOfGoodsYesNo,
+        helper.metric,
+        helper.quantity,
+        helper.lineItemNumberYesNo,
+        helper.lineItemNumber,
+        helper.additionalInformationYesNo,
+        helper.additionalInformation
+      ).flatten
+
+      val section = {
+        phaseConfig.phase match {
+          case Phase.Transition =>
+            Section(
+              rows.tail
+            )
+          case Phase.PostTransition =>
+            Section(
+              rows
+            )
+        }
+      }
```

Comment:
This looks fine but could we make it a little more clear which field we are removing? Maybe have two lists like:

val transitionRows = Seq(helper.attachToAllItems) ++ Seq(...)
val postTransitionRows = Seq(...)

Will also help if we need to add anything else. This is personal preference though, more than happy to keep if anyone else disagrees :)

Reply:
I think your suggestion makes sense, will update this.